2019/OSCON/the-inner-source-learning-path/proposal.md

```diff
@@ -0,0 +1,89 @@
+Proposal to be given via https://conferences.oreilly.com/oscon/oscon-or/user/proposal/propose/cfp/694
+
+# Title
+
+The Inner Source Learning Path
+
+# One-sentence description of your presentation
+
+Learn key roles and principles for effective inner source and how you can share them with others.
+
+# Topic
+
+Sustainable Open Source
+
+# Abstract
+
+Inner source is the application of open source methodologies to company-internal software.
```

Comment:
If I were Klaas, my comment would be: s/methodologies/methods. _Methodology_ is the field of study of methods. I think he has a point.

Reply:
Updated.